src/ionosphere/waccmx/edyn_geogrid.F90

```diff
@@ -16,7 +16,8 @@ module edyn_geogrid
     nlat,    & ! number of latitudes
     nlon,    & ! number of longitudes
     nlev,    & ! number of midpoint levels
-    nilev      ! number of interface latitudes
+    nilev,   & ! number of interface latitudes
```

Comment:
The comment should read "number of interface levels".  I know this is not your mistake.

Reply:
I might have looked at that comment sideways :)
Or maybe I need to start paying attention when I type :(